plutus-tx-plugin/test/IsData/deconstructData.plc.golden

```diff
@@ -1,6 +1,24 @@
 (program
   (let
     (nonrec)
+    (termbind
```

Comment:
Why has the inlining got worse in a bunch of these tests?

Reply:
We disabled simplifier on `plutus-tx-plugin` tests by setting `max-simplifier-iterations=0` in `plutus-tx-plugin.cabal`.